frontend/src/services/Blockchain/solana/BlockchainServiceSolana.ts

```diff
@@ -168,10 +174,13 @@ export class BlockchainServiceSolana
           value: Long.fromString(keysignPayload.toAmount),
           memo: keysignPayload?.memo,
         }),
-        recentBlockhash: recentBlockHash,
+        recentBlockhash: newRecentBlockHash,
         sender: keysignPayload.coin.address,
         priorityFeePrice: TW.Solana.Proto.PriorityFeePrice.create({
-          price: Long.fromString(priorityFee),
+          price: Long.fromString(priorityFeePrice.toString()),
+        }),
+        priorityFeeLimit: TW.Solana.Proto.PriorityFeeLimit.create({
+          limit: Number(20000),
```

Comment:
we don't set prioirtyfeelimit in IOS as far as I understand , won't this cause the problem with signing?

Reply:
I've added in the next PR to IOS as well )